### ISSUE TITLE
Extend functions & notebook

### ISSUE THREAD

**Issue by vcheplygina** [tueimage/8qa01#1]
* Simple classification, for example k-NN
* Randomly sample train/test set, or cross-validation setup

**Comment by mphjoosten** [comment]
I have added knn_classifier which is run using a train, validation, test, cross-validation setup. Two K's are now tested on the validation set and the best K is chosen for predictions on the test set. The score metric i have now implemented is accuracy but i could change this to roc-auc if necessary. I could make more code 'from scratch' by adding functions, and add some visualizations if needed.

**Comment by vcheplygina** [comment]
Looks good!
